app/models/manageiq/providers/workflows/automation_manager/workflow.rb

```diff
@@ -1,7 +1,15 @@
-class ManageIQ::Providers::Workflows::AutomationManager::Workflow < Workflow
+class ManageIQ::Providers::Workflows::AutomationManager::Workflow < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptPayload
```

Comment:
NOTE the embedded_ansible provider calls these `Playbooks` [[ref]](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/embedded_ansible/automation_manager/playbook.rb) so kept with the theme and called these Workflows